card/lib/card/format/error.rb

```diff
@@ -1,6 +1,40 @@
 class Card
   class Format
     module Error
+      def ok? task
+        task = :create if task == :update && card.new_card?
+        @ok ||= {}
+        @ok[task] = card.ok? task if @ok[task].nil?
+        @ok[task]
+      end
+
+      def anyone_can? task
+        return false unless task.is_a? Symbol
```

Comment:
also just for caching to optimize multiple views in a single format. See `active_cache_permissible?`